src/Layout/redist/minimumMSBuildVersion

```diff
@@ -1 +1 @@
-17.2.0
+17.3.0
```

Comment:
It would be nice if 17.4.0 was minimum msbuild.

Reply:
Unfortunately, we can't do that as we have a significant number of customers that want to update their SDK but not their Visual Studio version and still target downlevel. There is also sometimes a delay in pipelines on how quickly they can get Visual Studio updated on release date (for 5.0 for example, actions was delayed by ~3 weeks as I recall where anyone trying to use the 6.0 sdk was blocked).

See more details on this model here: https://docs.microsoft.com/en-us/dotnet/core/porting/versioning-sdk-msbuild-vs#targeting-and-support-rules  We'll update to 17.4 with 7.0.200 giving customers one quarter to update their Visual Studio version.